app/javascript/mastodon/containers/domain_container.js

```diff
@@ -21,6 +22,8 @@ const mapDispatchToProps = (dispatch, { intl }) => ({
       message: <FormattedMessage id='confirmations.domain_block.message' defaultMessage='Are you really, really sure you want to block the entire {domain}? In most cases a few targeted blocks or mutes are sufficient and preferable.' values={{ domain: <strong>{domain}</strong> }} />,
       confirm: intl.formatMessage(messages.blockDomainConfirm),
       onConfirm: () => dispatch(blockDomain(domain)),
+      destructive: true,
+      ...confirmDomainBlock && { passphrase: 'block' },
```

Comment:
Maybe this should be translatable?
Also I don't find the `...expression` form very readable with a condition.